ariadne/cardano/src/Ariadne/Wallet/Backend/KeyStorage.hs

```diff
@@ -262,20 +259,16 @@ newAccount
   -> IO ()
 newAccount pwl WalletFace{..} walletSelRef walletRef mbAccountName = do
   walletDb <- pwlGetDBSnapshot pwl
-  (hdrId, accList) <- second toList <$>
+  (hdrId, _) <- second toList <$>
       resolveWalletRefThenRead walletSelRef walletRef walletDb readAccountsByRootId
```

Comment:
Since the second item is ignored, you don't need `resolveWalletRefThenRead`, `resolveWalletRef` is enough.